resources/views/form_fields/select.blade.php

```diff
@@ -0,0 +1,22 @@
+<x-input.select
```

Comment:
This component doesn't exist, see: https://github.com/rapidez/core/tree/master/resources/views/components